web/beacon-app/src/features/onboarding/components/Step.tsx

```diff
@@ -20,11 +21,11 @@ const Step = () => {
       stepContent = <UserPreferenceStep />;
       break;
     default:
-      stepContent = <OrganizationStep />;
+      stepContent = <WorkspaceStep />;
```

Comment:
`WorkspaceStep` is the 2nd step of the onboarding process. Should step 1 be the default step?

Reply:
I put this just to be able to see the step 2 since the step buttons logic is not ready yet